allennlp/semparse/contexts/atis_sql_table_context.py

```diff
@@ -162,18 +123,19 @@ def initialize_valid_actions(self) -> Dict[str, List[str]]:
             # Sequence represents a series of expressions that match pieces of the text in order.
             # Eg. A -> B C
             if isinstance(rhs, Sequence):
-                valid_actions[key].add(format_action(key, " ".join(rhs._unicode_members()))) # pylint: disable=protected-access
+                valid_actions[key].add(format_action(key, " ".join(rhs._unicode_members()), # pylint: disable=protected-access
```

Comment:
We can potentially move this function to somewhere more generic like `sql_context_utils` since isn't doing dataset specific; not necessarily in this PR.